zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -202,6 +222,22 @@ public CollectionDescription create(HttpServletRequest request, HttpServletRespo
         return collection.getDescription();
     }
 
+    private void requireEditPermission(Session session) throws UnauthorizedException {
+        boolean canEdit = false;
+        try {
+            canEdit = permissionsService.canEdit(session);
+        } catch (IOException ex) {
+            throw new UnauthorizedException("You are not authorised to edit collections.");
```

Comment:
Same here - update the message to reflect the broader error, include the session ID and the original exception in the constructor params.